json2junit/testdata/TestRun/chatty/chatty.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+<testsuite name="cmd/go" tests="2" failures="0" skipped="0" time="0.011" timestamp="2025-02-07T14:43:16+01:00">
```

Comment:
It might be worth adding a tiny happy path test json that's easy for a person to read in full (without truncation by default) (maybe also easier to debug into). `chatty.json` is the smallest one, but it's still intentionally not short.

Here's one: https://github.com/microsoft/go-infra/commit/828f05cbf4600e119698ed3e3fc3337895935b69

Reply:
Good point. I've replaced all the lengthy samples with shorter and focused ones.